From e271879b3b455dad5b2914f6ce7e1152864f5ba9 Mon Sep 17 00:00:00 2001
From: Nikita Poltorapavlo <Nikita_Poltorapavlo2@cable.comcast.com>
Date: Mon, 29 Nov 2021 18:56:31 +0200
Source: COMCAST
Upstream-Status: Pending
Subject: [PATCH] RDKTV-11792: increase retry count

Reason for change: Internet precondition is never met
if the network connection is not up in first 200s.
Test Procedure: Start with network off, connect
after 5-6 min.
Risks: Low
Signed-off-by: Nikita Poltorapavlo <npoltorapavlo@productengine.com>
---
 LocationSync/LocationSync.config       | 2 +-
 LocationSync/LocationSyncPlugin.json   | 2 +-
 LocationSync/doc/LocationSyncPlugin.md | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/LocationSync/LocationSync.config b/LocationSync/LocationSync.config
index ea7584a..1478bcc 100644
--- a/LocationSync/LocationSync.config
+++ b/LocationSync/LocationSync.config
@@ -4,7 +4,7 @@ set (preconditions Network)
 
 map()
     kv(interval 10)
-    kv(retries 20)
+    kv(retries 60)
     kv(source ${PLUGIN_LOCATIONSYNC_URI})
 end()
 ans(configuration)
diff --git a/LocationSync/LocationSyncPlugin.json b/LocationSync/LocationSyncPlugin.json
index d016e8f..fca6178 100644
--- a/LocationSync/LocationSyncPlugin.json
+++ b/LocationSync/LocationSyncPlugin.json
@@ -23,7 +23,7 @@
                     "retries": {
                         "type": "number",
                         "size": 8,
-                        "description": "Maximum number of request reties to the Location Server (default:20)."
+                        "description": "Maximum number of request reties to the Location Server (default:60)."
                     },
                     "source": {
                         "type": "string",
-- 
2.25.1

